library/Notifications/Widget/ItemList/EventList.php

```diff
@@ -37,6 +39,9 @@ protected function init(): void
         $this->on(self::ON_ASSEMBLED, function () {
             ObjectsRendererHook::load();
         });
+
+        $this->getAttributes()
+            ->set('data-icinga-detail-url', (string) Url::fromPath('notifications/event'));
```

Comment:
Same change as for `ContactGroupList`.
Remember to introduce null values to `Links::event()` as well.